### DISCUSSION TITLE
Switching from spotipy to spotipy2

### DISCUSSION THREAD

**Discussion by ehan03** [CyanBook/spotipy2#9] [Q&A] [answered]
How easy is it to use spotipy2 for existing code that currently uses the spotipy library? I'm working on a web app but ran into request timeout issues after deploying to Heroku because the api calls were taking too long. Thanks!

**Comment by CyanBook** [comment, marked as answer]
Hi! Yeah, converting the code should be pretty easy, but please be aware that not every method in spotipy is available in spotipy2. Let me know for any trouble.

**Reply by ehan03** [reply]
Hey! Thanks so much for the reply. What are some major methods that are currently unavailable in spotipy2 (just for example)? Adding on to that, do you plan to continue expanding spotipy2's functionality to eventually cover every method? Have a great day!

**Reply by CyanBook** [reply]
I plan to add everything available in the Web API, but I'm alone and it's not easy. I'll try my best though!

**Reply by ehan03** [reply]
No worries, really appreciate the work you’ve done so far. Definitely gonna be keeping my eye on this repo. Godspeed!

**Reply by CyanBook** [reply]
If you want to stay updated, please click the "Watch" button at the top of the page. A very big update is coming soon!
